ostelco-ios-client/ViewControllers/Country/CoverageViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  CoverageViewController.swift
+//  ostelco-ios-client
+//
+//  Created by Samuel Goodwin on 10/1/19.
+//  Copyright © 2019 mac. All rights reserved.
```

Comment:
This isn't strictly true :-). The file headers should reflect that this is red otter software.

Reply:
our IDE (xcode) creates this, should be configurable though (I assume). I'm creating a task in notion to remember this.